dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -500,12 +500,13 @@ def test_column_stats_profile_compiler_stats_diff(self):
                 "categories": [["1"], ["9"], ["10"]],
                 "gini_impurity": 0.06944444444444448,
                 "unalikeability": 0.16666666666666663,
-                "categorical_count": {"9": -1, "1": [1, None], "10": [None, 1]},
+                "categorical_count": {"9": -1, "1": 1, "10": -1},
                 "chi2-test": {
                     "chi2-statistic": 2.1,
                     "df": 2,
                     "p-value": 0.3499377491111554,
                 },
+                "psi": 0.009815252971365292
```

Comment:
PSI value for `ColumnStatsProfileCompiler` that is identifying the test data as categorical... therefore needs a PSI value for `expected_diff`